web-components/src/components/tabs/Tabs.ts

```diff
@@ -62,6 +68,29 @@ export namespace Tabs {
     @internalProperty() private tabsViewportDataList: TabsViewportDataList = [];
     @internalProperty() private tabsFilteredAsVisibleList: Tab.ELEMENT[] = [];
     @internalProperty() private tabsFilteredAsHiddenList: Tab.ELEMENT[] = [];
+    @internalProperty() private noTabsVisible = false;
+
+    @property({ type: Number }) delay = 0;
```

Comment:
Is every instance of @property something that we want to be accessible from outside the component as an attribute or property? Are these expected to be set by users or just used by higher-order components?

Reply:
Yes these can be given by the user if they want to customize the draggable tabs otherwise this is not being used by any other higher-order components